rust-toolchain

```diff
@@ -1,5 +1,4 @@
 [toolchain]
-channel = "1.77.0"
```

Comment:
Why deleting this? I don't mind upgrading, but for tracking the versions used, I think this should be here. And in any case, if we're taking it out, it shouldn't be in this PR.

Reply:
Yes, agreed. Unintentional.

Reply:
I'd just avoid upgrading the toolchain in this repo altogether. To keep the scope clear.